content/customize-gnome.md

```diff
@@ -18,117 +18,141 @@ section: software
 
-## Dash to Dock
+Pop!\_OS includes the following extensions by default. In most cases, these extensions should be left **enabled**.
 
-[Dash to Dock](https://extensions.gnome.org/extension/307/dash-to-dock/)
+### [Cosmic Dock](https://github.com/pop-os/cosmic-dock)
 
-![dash-to-dock](/images/customize-gnome/dash-to-dock.png)
+A fork of Dash to Dock customized for Pop!\_OS.
 
-This extension brings a launcher similar to the Unity launcher on the left side, or the macOS launcher on the bottom.  It is high customizable and allows for different locations, styles, sizes, and layouts.  This is included by default in Ubuntu 18.04 and can be added to Pop!_OS.
+![cosmic-dock-close](/images/customize-gnome/cosmic-dock-close.png)
 
----
+**NOTE:** It is **not** recommended to have both Cosmic Dock and Dash-t0-Dock installed an enabled at the same time. User Interface (UI) bugs will occur. Choose one or the other.
 
-## Backslide
+### [Cosmic Workspaces](https://github.com/pop-os/cosmic-workspaces)
 
-[Backslide](https://extensions.gnome.org/extension/543/backslide/)
+A fork of this [extension](https://github.com/RensAlthuis/vertical-overview) designed to return the GNOME workspaces to a vertical layout rather than the horizontal view introduced in GNOME 40.
 
-![backslide](/images/customize-gnome/backslide.png)
+Workspace behaviors can be edited in Settings.
 
-This extension is a wallpaper slideshow for your desktop.  After choosing a folder of pictures, they will be rotated automatically at a predefined time period.
+![workspaces-and-settings](/images/customize-gnome/workspaces-and-settings.png)
 
----
+### [Desktop Icons NG (DING)](https://gitlab.com/rastersoft/desktop-icons-ng)
 
-## Window List
+Since Desktop icons have been deprecated in mainline GNOME, Pop!_OS includes a GNOME extension to return this feature.
 
-[Window List](https://extensions.gnome.org/extension/602/window-list/)
+### [Pop COSMIC](https://github.com/pop-os/cosmic)
 
-![window-list](/images/customize-gnome/window-list.png)
+![cosmic-dock](/images/customize-gnome/cosmic-dock.png)
 
-This extension adds a window list to the bottom of the monitors.  This shows what windows are open, and allows for them to be minimized, maximized, and closed from the bar, similar to Microsoft Window's window list.
+Computer Operating System Main Interface Components **(C.O.S.M.I.C.)**
 
----
+COSMIC is the name for the main shell components in Pop_Shell (separate from the tiling and window-management components). It makes the following changes to the standard GNOME Shell environment:
 
-## Gnomenu
+- Dock
+- Multi-Monitor improvements
+- Configuration options
+- Separated Workspaces overview from Applications.
 
-[Gnomenu](https://extensions.gnome.org/extension/608/gnomenu/)
+Most components can be configured to fit the user's workflow and preferences, with two main presets for both keyboard-focused and mouse-focused navigation and use.
```

Comment:
And use? Maybe too long?

Reply:
Adjusted wording a bit and made the sentence shorter.